src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
+  @ApiProperty({
+    example: 'username',
```

Comment:
PullProofAI

Consider adding a validation decorator for the `name` field to ensure it meets specific criteria like minimum and maximum length.



<!-- This is an auto-generated comment by PullProofAI -->